storybookjs/storybook
# UI: Reset layout properties when switching stories

Issue: #10641 

## What I did
Because there is a merge operation with styles when switching stories, when changing from centered to padded, for instance, the properties from the layout centered that were not present in padded (such as justify-content) would still be there, therefore making the next element be centered by mistake. The fix is to reset those properties. 

![Kapture 2020-05-04 at 22 18 10](https://user-images.githubusercontent.com/1671563/81009923-c5326500-8e55-11ea-8b55-d9436e317b91.gif)

## How to test
1 - Checkout this branch
2 - Run storybook official example
3 - Filter for layout stories and check there
